analyses/fusion_filtering/03-Calc-zscore-annotate.R

```diff
@@ -60,11 +64,13 @@ standardFusionCalls<-opt$standardFusionCalls
 expressionMatrix<-opt$expressionMatrix
 zscoreFilter<- opt$zscoreFilter
 saveZscoredMatrix<-opt$saveZscoredMatrix
-gtexMatrix<-opt$normalExpressionMatrix
+clinicalFile <- opt$clinicalFile
+cohortInterest<-unlist(strsplit(opt$cohortInterest,","))
+gtexMatrix<-unlist(strsplit(opt$normalExpressionMatrix,","))
 
 print(gtexMatrix)
 
-ZscoredAnnotation<-function(standardFusionCalls=standardFusionCalls,zscoreFilter=zscoreFilter,saveZscoredMatrix=saveZscoredMatrix,normData=normData,expressionMatrix=expressionMatrix){
+ZscoredAnnotation<-function(standardFusionCalls=standardFusionCalls,zscoreFilter=zscoreFilter,saveZscoredMatrix=saveZscoredMatrix,expressionMatrix=expressionMatrix){
```

Comment:
I believe you can retain the `normData` and add a list of samples in each cohort `cohort_BSids` as param to this function and which can help filter `standardFusionCalls` and `expressionMatrix` for list of cohort_BSids and use the associated `normData`. Without adding more code to the function right? 

We can then the function can be called and rbind like:

```

GTExZscoredAnnotation_filtered_fusions_list <- lapply(cohortInterest,
  function(x) {
cohort_BSids <- read.delim(clinicalFile, header = TRUE, sep = "\t", stringsAsFactors = FALSE) %>%
     # uses each value x in cohortInterest 
      filter(cohort == x) %>%
      filter(experimental_strategy == "RNA-Seq") %>%
      filter(sample_type == "Tumor") %>%
      pull("Kids_First_Biospecimen_ID")

# per cohort zscore annotation 
GTExZscoredAnnotation_filtered_fusions<- ZscoredAnnotation(standardFusionCalls ,zscoreFilter,normData=normData_brain,cohort_BSids= cohort_BSids, expressionMatrix = expressionMatrix)
return(GTExZscoredAnnotation_filtered_fusions)
}

GTExZscoredAnnotation_filtered_fusions <- do.call(rbind,GTExZscoredAnnotation_filtered_fusions_list ) 
```